dbms/src/Storages/Transaction/KVStore.cpp

```diff
@@ -338,6 +338,15 @@ bool KVStore::tryFlushRegionData(UInt64 region_id, bool try_until_succeed, TMTCo
 {
     auto region_task_lock = region_manager.genRegionTaskLock(region_id);
     const RegionPtr curr_region_ptr = getRegion(region_id);
+    if (curr_region_ptr == nullptr)
```

Comment:
Should we also handle can not get region in `needFlushRegionData`?

Reply:
This function is not used till now, it may not return true for this case. I will add a TODO here.